api/tools/src/main/resources/bigquery/es_person.sql

```diff
@@ -18,6 +18,7 @@ SELECT
   measurement_concept_ids,
   measurement_source_concept_ids,
   visit_concept_ids,
+  case when death.person_id is null then false else true end as is_deceased,
```

Comment:
Can't this be rewritten as `death.person_id is null as is_deceased` ?

Reply:
Good catch. Actually it needs to be death.person_id is not null as is_deceased.